AliceFaberAcmeDemo/controllers/faber-controller/FaberController/Startup.cs

```diff
@@ -30,13 +30,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddServerSideBlazor();
             services.AddHttpClient<FCAgentService>(c =>
             {
-                var agentUrl = Environment.GetEnvironmentVariable("FABER_AGENT_HOST");
+                var agentUrl = "host.docker.internal";
```

Comment:
Should this be a default value to fall-back to if `FABER_AGENT_HOST` is unset, rather than replacing things completely? It also looks like the variable name might have changed, in `docker-compose.yaml` it looks like it might be `AGENT_HOST`.